src/content/docs/alerts/get-notified/muting-rules-suppress-notifications.mdx

```diff
@@ -17,78 +17,88 @@ Alerts sends out timely notifications when your system is having problems. Somet
+Muting rules override specific incidents. They don't disable existing policies or conditions. This allows you to mute incidents from specific entities that may be covered by a policy or condition that covers a large number of entities. This also keeps you from having to over-mute your monitoring when you are performing maintenance on a subset of your system.
 
-## Create a muting rule [#create]
+The following table describes how the alerts incident lifecycle is affected by muted incidents:
 
-<Callout variant="tip">
-  Before creating muting rules, you'll need to [create policies](/docs/alerts/new-relic-alerts/configuring-alert-policies/create-edit-or-find-alert-policy) and [conditions](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-alert-conditions/) that generate [notifications](/docs/alerts-applied-intelligence/notifications/intro-notifications/).
-</Callout>
+<table>
+  <thead>
+    <tr>
+      <th>
+        IF
+      </th>
 
-To create a muting rule, go to <DNT>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Alerts > Muting rules**</DNT> and click <DNT>**+ Add a rule**</DNT>. Enter a name and a description (optional) for the muting rule, and select the account to which the rule will apply.
+      <th>
+        THEN
+      </th>
+    </tr>
+  </thead>
 
-Next, build the incidents filter. You can use a subset of [incident event attributes](/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/understand-technical-concepts/incident-event-attributes). Choose an attribute, an [operator](#sub-conditions), and a value. These are the attributes: `accountId`, `conditionId`, `conditionName`, `conditionType`, `entity.guid`, `nrqlEventType`, `nrqlQuery`, `policyId`, `policyName`, `product`,`runbookUrl` (as `conditionRunbookUrl`), `tags.<NAME>`, and `targetName`). Values are compared against one of your incident attributes, such as an alerts policy ID or a condition name.
-Click <DNT>**Add another condition**</DNT> if you want to include more filters.
+  <tbody>
+    <tr>
+      <td
+        colSpan={3}
+        style={{ backgroundColor: "rgb(238, 238, 238)" }}
+      >
+        <DNT>**Event**</DNT>: Issue is activated
+      </td>
+    </tr>
 
-<img
-  title="Muting rule edit screen"
-  alt="Muting rule edit screen"
-  src="/images/accounts_screenshot-full_violation-filter.webp"
-/>
+    <tr>
+      <td>
+        An issue is activated because of an incident that is **not** muted
+      </td>
 
-<figcaption>
-  <DNT>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Alerts > Muting rules**</DNT>: You can create complex muting rules to target a small or large set of unwanted notifications.
-</figcaption>
+      <td>
+        Notifications for this issue will be sent.
+      </td>
+    </tr>
 
-## Schedule a muting rule [#schedule-muting-rule]
+    <tr>
+      <td>
+        An issue is activated due to an incident that <DNT>**is**</DNT> muted
```

Comment:
remove the <DNT> tag and add a punctuation mark. I assume <DNT> tags are used only for UI elements

Reply:
This was like this before. I'm going to ask the SME, just in case....